src/components/InfoTooltip/index.tsx

```diff
@@ -1,32 +1,37 @@
 import { FloaterPlacement, Icon, Tooltip } from "@stellar/design-system";
 import "./styles.scss";
 
+const ICONS = {
+  info: <Icon.Info />,
+  check: <Icon.CheckCircle color="green" />,
+};
+
 interface InfoTooltipProps {
   children: React.ReactNode;
   infoText: React.ReactNode;
   placement?: FloaterPlacement;
   hideTooltip?: boolean;
+  icon?: "info" | "check";
 }
 
 export const InfoTooltip = ({
   children,
   infoText,
   placement = "right",
   hideTooltip = false,
+  icon = "info",
 }: InfoTooltipProps) => {
   if (hideTooltip) {
     return <>{children}</>;
   }
 
+  const iconComponent = ICONS[icon] ?? ICONS.info;
```

Comment:
We're setting `icon` default value to `info`, so we should always get something here.

```suggestion
  const iconComponent = ICONS[icon];
```